examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -11,9 +11,13 @@
 #
 # License: BSD (3-clause)
 
+import numpy as np
 import mne
 from mne import io
 from mne.datasets import sample
+import matplotlib.pyplot as plt
```

Comment:
since you're digging around in here anyway, might as well move the `mpl` import out from the middle of these `mne` imports and up near the `numpy` import instead